src/main/java/frc/trigon/robot/commands/commandfactories/GeneralCommands.java

```diff
@@ -22,12 +22,16 @@ public static Command withoutRequirements(Command command) {
     }
 
     /**
-     * @return a command that toggles between the SWERVE's default command, from field relative to self relative
+     * A command that toggles between the SWERVE's default command, from using Joystick oriented rotation to using normal rotation.
```

Comment:
Should start with `creates a command`, and then you can simply type `the command` in @ return